.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: required
 before_install:
 - sudo apt-get update -qq
 - sudo apt-get install -y iputils-ping
+- go get -a -x -installsuffix cgo -ldflags '-d -s -w' github.com/mattn/gom
```

Comment:
would you mind removing all the "-x" from these, or would you rather keep it *that* verbose